private/dev/Dashboard.js

```diff
@@ -80,7 +100,7 @@ export default () => {
       proudlyDisplayPoweredByUppy: true,
       note: `${JSON.stringify(restrictions)}`,
     })
-    // .use(GoogleDrive, { target: Dashboard, companionUrl: COMPANION_URL, companionAllowedHosts })
+    .use(GoogleDrive, { target: Dashboard, companionUrl: COMPANION_URL, companionAllowedHosts, ...getCompanionKeysParams('GOOGLE_DRIVE') })
```

Comment:
@arturi should `RemoteSources` support this too? Otherwise you need both the regular plugin and remote sources.

Reply:
maybe we should do that in a different PR anyways?

Reply:
For sure, just wanted to make Artur aware.